templates/land-see/stories.amp.html

```diff
@@ -43,36 +81,131 @@
     <amp-animation id="subscribeBGParallax" layout="nodisplay">
       <script type="application/json">
         {
-        "duration": "1s",
-        "fill": "both",
-        "direction": "alternate",
-        "animations": [
-          {
-            "selector": ".land-see-subscribe-bg",
-            "keyframes": [
-              { "transform": "translateY(calc(height('.land-see-subscribe') * -0.4))" }
-            ]
-          }
-        ]
-      }
+          "duration": "1s",
```

Comment:
replace tabs with spaces

Reply:
This should be the same as `land-see.amp.html`, featuring spaces.